ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Nagios_Plugins/tasks/additional_plugins/check_docker_overlay2_size.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# Nagios plugin to check the size of /var/lib/docker/overlay2
+# Allows passing thresholds as parameters
+
+# Default thresholds (in GB)
+DEFAULT_WARN_THRESHOLD=30
```

Comment:
I'd prefer to go lower if possible as an initial pass so that we see which ones are hitting this level. I don't think there's a lot we're doing on these systems that should push it above 20 in normal use, and if there is it would be good to collect data on it before increasing the number
```suggestion
DEFAULT_WARN_THRESHOLD=20
```

Noting that the current x64 dockerhost is currently sitting with 
```
TYPE            TOTAL     ACTIVE    SIZE      RECLAIMABLE
Images          72        11        64.79GB   40.02GB (61%)
Containers      12        11        32.89GB   410.4MB (1%)
Local Volumes   2         0         0B        0B
Build Cache     430       0         1.135GB   1.135GB
```
So it'll trigger a `critical` threshold immediately. I suspect it's going to make sense to have separate thresholds for `dockerhost` systems and other test machines which are running docker, but I think 20/40 is likely a good place to start for this.